bears/vcs/git/GitCommitBear.py

```diff
@@ -27,7 +27,7 @@ def run(self,
             shortlog_trailing_period: bool=None,
             shortlog_imperative_check: bool=True):
         """
-        Checks the current git commit message at HEAD.
+        Check the current git commit message at HEAD.
```

Comment:
yep, out of scope for this PR

If this is a new codestyle we're introducing then we need to make an issue to do this and add documentation appropriately in codestyle docs.

Not sure if there was some discussion about adding this to our codestyle (Didn't find any issue)